### ISSUE TITLE
Language Phrase for third Party App

### ISSUE THREAD

**Issue by Bjoern3003** [PHPfox-Official/phpfox-v4-feature-requests#797]
Hi, we have problems with understand of Phrases.

A Plugin, for example socialbridge add own Language Phrases. All Phrases where added, but not with his own module_id. So this Plugin is call a Phrase like this: {_p var='socialbridge.to'}

He is getting Phrase from Core.
![15032019100911](https://user-images.githubusercontent.com/13675579/54517485-1e91b900-4962-11e9-9f2f-cd70eebe1fd7.jpg)

So if I change this Phrase for socialbridge, all other sections, where this Phrase is used, is also changed. This is not, what we want. I think, there is a Problem by importing the Phrase, that the importer is not set the module_id?

I hope, you understand, what I mean :-)

Thanks

**Comment by harrison05** [comment]
Yes, we removed module_id in phrases before. If 3rd wants to use their own phrases, they should add prefix module_id on it. Example in phrase.json: "socialbridge_to": "To"

Regards.

**Comment by Bjoern3003** [comment]
But why did you not support die modul_id as prefix?

**Comment by harrison05** [comment]
Because we want to reduce duplicate phrases between apps and it helps Admins don't have to search and translate a phrase many times.

**Comment by Bjoern3003** [comment]
But sometimes, translation in other Languages need on any places another phrase, because it means anything else and need other words for it.
